### ISSUE TITLE
Different font-sizes in heading for Sec 2 (top) and monospace fonts (Sec 1.1) jarring

### ISSUE THREAD

**Issue by akatlas** [rfc-format/draft-iab-rfc-css-bis#6]
I found the change from the expected heading size to be jarring.  Perhaps it's intentional, but
for the monospace font in Sec 1.1 and the whole heading title (top) for Section 2, it looks odd to me.

**Comment by martinthomson** [comment]
I agree.  Usually monospace fonts need a size adjustment to keep them looking natural, but `<code>` and `<tt>` are defined as fixed size.    Defining them with proportional sizes (90% for instance) fixes that problem.

**Comment by martinthomson** [comment]
I agree.  Usually monospace fonts need a size adjustment to keep them looking natural, but `<code>` and `<tt>` are defined as fixed size.    Defining them with proportional sizes (90% for instance) fixes that problem.
